io/EptReader.cpp

```diff
@@ -125,8 +91,8 @@ BOX3D EptReader::Args::bounds() const
     const BOX2D b(m_bounds.to2d());
     if (b.empty())
     {
-        double mn((std::numeric_limits<double>::lowest)());
-        double mx((std::numeric_limits<double>::max)());
+        const double mn((std::numeric_limits<double>::lowest)());
```

Comment:
Wouldn't this cause an issue if you had legitimate negative coordinates but did not specify bounds?

Reply:
Not sure I understand this one - can you explain further?  The logic here is intended to set the query boundaries to `(-Inf, -Inf, -Inf, Inf, Inf, Inf)`, if a query boundary is not supplied by the user.

Reply:
I've just confused min and lowest again.